setup.py

```diff
@@ -34,6 +34,7 @@
             "codecov",
             "black==20.8b1",
             "pydocstyle==5.1.1",
+            "wheel",
```

Comment:
I added wheel locally so that the developers can build on their machines, if necessary. Otherwise, there is an error message (see [this StackOverflow question](https://stackoverflow.com/questions/34819221/why-is-python-setup-py-saying-invalid-command-bdist-wheel-on-travis-ci)).